charts/README.md

```diff
@@ -7,124 +7,18 @@
 - Prerequisites
   - Kubernetes version is between 1.14 and 1.20.
   - The version of Prometheus Operator is v0.49.0.
-- Apply for a cluster from [TKE](https://cloud.tencent.com/product/tke) or install [minikube](https://minikube.sigs.k8s.io/docs/start/).
+- Apply for a cluster from [TKE](https://cloud.tencent.com/product/tke) or install [Minikube](https://minikube.sigs.k8s.io/docs/start/).
```

Comment:
It is recommended to also explain that kstone supports deployment in various cloud vendors and bare k8s cluster environments, and only the corresponding ingress rules need to be configured.